client.go

```diff
@@ -159,6 +158,10 @@ func (c *Client) sendScalingMsg(data *logMetrics) {
 		"prefix": *data.prefix,
 	}).Debug("sendScalingMsg")
 
+	for _, v := range data.events {
+		c.SimpleEvent("Heroku app: "+*data.app, v)
```

Comment:
APPNAME we have in `*data.app` and that why I can add into event. And heroku is good to know, you can have events from many systems.